gradle/libs.versions.toml

```diff
@@ -45,17 +45,18 @@ jcodec = "0.2.5"
 kotlin17 = "1.7.10"
 kotlin18 = "1.8.22"
 kotlin19 = "1.9.24"
-kotlin = "1.9.24" # Don't forget to set kotlinNative the same version
+kotlin = "2.1.0" # Don't forget to set kotlinNative the same version
 kotlinBenchmark = "0.4.11"
-kotlinGradlePluginAnnotations = "1.9.24"
-kotlinGradlePluginApi = "1.9.24"
-kotlinNative = "1.9.24"
+kotlinGradlePluginAnnotations = "2.1.0"
+kotlinGradlePluginApi = "2.1.0"
+kotlinNative = "2.1.0"
```

Comment:
Do we still need it as a separate property?

Reply:
Just checked it. It's still needed. It's used implicitly in build scripts

Reply:
We can change build scripts

Reply:
fixed